docs/api-reference/aggregation-layers/heatmap-layer.md

```diff
@@ -121,6 +128,10 @@ Method called to retrieve the position of each point.
 
 Method called to retrieve weight of each point. By default each point will use a weight of `1`.
 
+## Limitations
+
+The `HeatmapLayer` performs aggregation on the GPU. It requires the WebGL context to support rendering to a float texture. This feature is generally supported in evergreen desktop browsers, but not iOS Safari. On iOS, the layer falls back to a low-precision mode.
```

Comment:
Start by bullet listing the affected platform and then the detailed technical reasons 
- iOS: low-precision mode will be used (because iOS Safari does not support floating point textures).

- What are the consequences of falling back to low-precision mode (slightly worse graphics, problem when zoomed in etc). The text at the top says "not supported" this says "lower precision".

- Another bullet mentioning the windows limitation discussed above?

Reply:
The Angle issue was fixed in 7.3. I will remove the remark from the doc.